packages/ckeditor5-alignment/docs/features/text-alignment.md

```diff
@@ -11,6 +11,16 @@ The {@link module:alignment/alignment~Alignment} feature enables support for tex
 
 {@snippet features/text-alignment}
 
+## Related features
+
+There are more CKEditor 5 features that can help you organize your content:
+* {@link features/basic-styles Basic font styles} &ndash; The essentials, like **bold**, *italic* and others.
+* {@link features/headings Headings} &ndash; Divide your content into sections.
+* {@link features/title Document title} &ndash; Clearly divide your content into a title and body.
```

Comment:
Maybe instead of having three "divides" one after another, we could use "split", "break down" in others?